code/modules/cargo/packs.dm

```diff
@@ -800,6 +799,14 @@
 	crate_type = /obj/structure/closet/crate/secure
 	dangerous = TRUE
 
+/datum/supply_pack/science/research
+	name = "Machine Prototype Crate"
+	cost = 5000
```

Comment:
I've never played cargo so I don't know about the prices. I just gave it something that looked pretty high-ish when comparing to other items. Better suggestions for the cost?